dags/analyzer_helper/discord/discord_extract_raw_members.py

```diff
@@ -0,0 +1,41 @@
+from analyzer_helper.discord.extract_raw_member_base import ExtractRawMembersBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordExtractRawMembers(ExtractRawMembersBase):
+    def __init__(self, guild_id: str, platform_id: str):
+        """
+        Initialize the class for a specific guild and platform
+        """
+        self.guild_id = guild_id
+        self.platform_id = platform_id
+        self.client = MongoSingleton.get_instance().client
+        self.guild_db = self.client[guild_id]
+        self.platform_db = self.client[platform_id]
+        self.guild_collection = self.guild_db["guildmembers"]
+        self.rawmembers_collection = self.platform_db["rawmembers"]
+
+    def extract(self, recompute: bool = False) -> list:
+        """
+        Extract members data
+        if recompute = True, then extract the whole members
+        else, start extracting from the latest saved member's `joined_at` date
+
+        Note: if the user id was duplicate, then replace.
+        """
+        members = []
+        if recompute:
+            members = list(self.guild_collection.find({}))
+        else:
+            # Fetch the latest joined date from rawmembers collection
+            latest_rawmember = self.rawmembers_collection.find_one(sort=[("joinedAt", -1)])
+            latest_joined_at = latest_rawmember["joinedAt"] if latest_rawmember else None
```

Comment:
on rawmembers_collection we have the filed in snake case format, so please use `joined_at` instead of `joinedAt` when querying the rawmembers collection. ref: https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=%22joined_at%22%3A%20datetime%2C

The camelCase was on guildmembers data structure. ref: https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=joinedAt%3A%20Date%20%7C%20null%2C